src/NewSubscription.m

```diff
@@ -202,6 +202,7 @@ -(IBAction)doSubscribe:(id)sender
 	// Validate the subscription, possibly replacing the feedURLString with a real one if
 	// it originally pointed to a web page.
 	rssFeedURL = [subscriptionModel verifiedFeedURLFromURL:rssFeedURL];
+	NSAssert(rssFeedURL != nil, @"No valid URL verified to attempt subscription !");
```

Comment:
Yes, but it always crashes the program whether or not a valid URL has been entered.